### PR TITLE
Order tabs correctly when they have openers

### PR DESCRIPTION
**Pull request by bbondy**

### CODE REVIEW

File: test/unit/state/frameStateUtilTest.js [hunk 1]
```diff
@@ -17,92 +17,6 @@ describe('frameStateUtil', function () {
     this.windowState = Immutable.fromJS(Object.assign({}, defaultWindowStore.toJS()))
   })
 
-  describe('cloneFrame', function () {
-    before(function () {
-      this.frameOpts = {
-        key: 1,
-        tabId: 2,
-        parentFrameKey: 3,
-        audioMuted: true,
```
Comment (bridiver) [inline test/unit/state/frameStateUtilTest.js]: I think this is the only one we need to handle and we can do that in electron -  https://github.com/brave/electron/issues/126

File: test/unit/state/frameStateUtilTest.js [hunk 2]
```diff
@@ -17,92 +17,6 @@ describe('frameStateUtil', function () {
...
-        canGoBack: true,
-        canGoForward: true,
-        icon: 'icon',
-        title: 'title',
-        isPrivate: true,
-        partitionNumber: 4,
-        themeColor: 'ffffff',
-        computedThemeColor: 'aaaaaa',
-        history: ['http://brave.com'],
```
Comment (bridiver) [inline test/unit/state/frameStateUtilTest.js]: do we still need this now that we have history methods on the webview/webcontents? Are we still using it anywhere?

**Comment by bridiver** [comment]
++

File: test/unit/state/frameStateUtilTest.js (hunk 2 above)
Comment (bbondy) [inline test/unit/state/frameStateUtilTest.js]: Possibly we're using it in about: pages.

Comment (bridiver) [inline test/unit/state/frameStateUtilTest.js]: can we replace that with tab/webview history now?

**Pull request merged**